robottelo/constants.py

```diff
@@ -307,8 +356,10 @@
     }
 }
 
-RHEL_6_MAJOR_VERSION = 6
-RHEL_7_MAJOR_VERSION = 7
+DISTRO_REPOS = {
+    # DISTRO_RHEL6: REPOS['rhel6'],
```

Comment:
Why is it commented?

Reply:
Because we have no rhel6 entry, but in the future as we have some tests in the backlog to use rhel6  (and we will add rhel6 repo data)

Reply:
will remove comment when create the test for https://github.com/SatelliteQE/robottelo/issues/5429